app/src/main/java/com/neudesic/myapplication/domain/models/DadJoke.kt

```diff
@@ -0,0 +1,30 @@
+package com.neudesic.myapplication.domain.models
+
+import com.google.gson.annotations.SerializedName
+
+
+data class DadJoke(
```

Comment:
I recommend the kotlin serializer from retrofit. Then you can annotate the class with `@Serializable`

Reply:
is that just so I dont have to do @SerializedName on each property?

Reply:
Yeah, it also handles the JSON differently. I believe it is _slightly faster_